src/main/webapp/style.css

```diff
@@ -365,6 +365,41 @@ table {
   padding-left: 35px;
 }
 
+#recommendation-list a {
+  color: black;
+  text-decoration: none;
+}
+
+#recommendation-list a img {
+  border-bottom: 1px solid #e7e7e9;
+  display: block;
+  padding-bottom: 15px;
+  margin: 0 auto 15px auto;
+  width: 150px;
+}
+
+#recommendation-list li {
+  border: 1px solid #e7e7e9;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 4px 4px 8px #e7e7e9;
```

Comment:
this should be up with the b's for alphabetical sorting

Reply:
fixed.